ex7/ex7.js

```diff
@@ -9,3 +10,59 @@
  * mostFrequentChar("a b c d e f") === " "
  * mostFrequentChar("") === null
  */
+
+function mostFrequentChar(str) {
+
+	// If str empty tmpChar will return NULL
+	var  winnerChar= null, index = 0,
+		topRepeat = { charValue: '', charCount: 0}, 
+		search = { charValue: '', charCount: 0}, 
+		doNotSearchList = [];
+
+	if (str != '') {
```

Comment:
I like to do this instead, because then the edge-case is being considered at the top, while the rest of the code remain simple (i.e. without the indentation):
```
if (str === '') {
  return null;
}
```